docs/src/main/asciidoc/smallrye-health.adoc

```diff
@@ -399,6 +399,34 @@ public class LivenessAsync implements AsyncHealthCheck {
 }
 ----
 
+== Health status change observers
+
+If you need to react to change of the health status of your application, the Smallrye Health
+extension provides a CDI event that can notify you about the individual health status changes.
+
+To observe the health status change, you can observe the `io.smallrye.health.api.event.HealthStatusChangeEvent` with the standard CDI observation mechanism:
```

Comment:
Is is okay or not to do blocking calls when handling these events?

Reply:
It's not guaranteed that it will always run on worker thread so better to mention. Thanks for the reminder.